types/src/access_path.rs

```diff
@@ -63,27 +63,27 @@ impl AccessPath {
         AccessPath { address, path }
     }
 
-    pub fn resource_access_vec(tag: StructTag) -> Vec<u8> {
+    pub fn resource_path_vec(tag: StructTag) -> Vec<u8> {
```

Comment:
`resource_access_path_vec` isn't too verbose to me..

Reply:
Just wanna make `resource` and `module` functions consistent.